src/meltano/core/project_settings_service.py

```diff
@@ -146,10 +138,10 @@ def meltano_yml_config(self):
     def environment_config(self):
         """Return current environment configuration in `meltano.yml`.
 
-        Returns:
-            Current environment configuration in `meltano.yml`
+        Raises:
+            NotImplementedError: Meltano config not supported in Environments.
         """
-        return self.config_service.current_environment_config
+        raise NotImplementedError("Meltano config not supported in Environments.")
```

Comment:
@kgpayne Would it make sense to remove these methods from the ABC?

Reply:
@edgarrmondragon I did think about that, and it is totally possible. My thought was a `NotImplementedError` with a clear description might be more insightful than a generic `AttributeError` if a future feature tries to access these properties on the wrong subclass of the `SettingsService` 🤔 That said, I am not sure how likely that is to happen, and agree its probably neater to just remove both properties from the ABC and the `ProjectSettingsService`. WDYT?

Reply:
An `AttributeError` might then indicate that the method has to be pushed up to the ABC, so yeah I'd vote for removing them from the ABC and the project-level implementation 👍